README.md

```diff
@@ -23,7 +23,8 @@ Use the CLI to create a new `memcached-operator` project:
 ```sh
 $ mkdir -p $GOPATH/src/github.com/example-inc/
 $ cd $GOPATH/src/github.com/example-inc/
-$ operator-sdk new memcached-operator
+$ export GO111MODULE=on
+$ operator-sdk new memcached-operator --repo github.com/example-inc/memcached-operator
```

Comment:
I do not think that the `--repo github.com/example-inc/memcached-operator` should be added here.

Reply:
You are right, let me update